tests/SimplePaginateTest.php

```diff
@@ -0,0 +1,52 @@
+<?php
+
+namespace Spatie\CollectionMacros\Test;
+
+use Illuminate\Support\Collection;
+
+class SimplePaginateTest extends TestCase
+{
+    /** @test */
+    public function it_provides_paginate_macro()
+    {
+        $this->assertTrue(Collection::hasMacro('simplePaginate'));
+    }
+
+    /** @test */
+    public function it_returns_relevant_context_information()
+    {
+        $p = (new Collection(['item1', 'item2', 'item3']))->simplePaginate(2, 'page', 2);
+
+        $this->assertEquals(2, $p->currentPage());
+        $this->assertTrue($p->hasPages());
+        $this->assertFalse($p->hasMorePages());
+        $this->assertEquals([2 => 'item3'], $p->items());
+        $pageInfo = [
+            'per_page' => 2,
+            'current_page' => 2,
+            'next_page_url' => null,
+            'prev_page_url' => '/?page=1',
+            'from' => 3,
+            'to' => 3,
+            'data' => [2 => 'item3'],
+            'path' => '/',
```

Comment:
`Paginator` returns `path` since [version 5.4.24](https://github.com/laravel/framework/releases/tag/v5.4.24) so this test is broken with earlier versions.